### ISSUE TITLE
Server doesn't operate in a streaming manner

### ISSUE THREAD

**Issue by cpiker** [das-developers/das2-tomcat-server#2]
It looks like each request is written to the single file, `/tmp/das2.server.qds`.  This means multiple requests can't be made to the same server at the same time.  This could be corrected by using various temp-name facilities, but there is a bigger issue with `das2server.doDataset()`.

All data look to be read and reduced to the temporary file above before data streaming begins.  One of the prime tenants of das2 servers is that data should be processed and written as a stream.

**Comment by jbfaden** [comment]
Yes, I saw that too.  This was more a proof-of-concept, and never was in production use (other than serving my temperatures).
